src/main/api/2/studio-api-2.yaml

```diff
@@ -2041,6 +2043,108 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /configuration/get_configuration:
+    get:
+      tags:
+        - configuration
+      summary: Get configuration content for site and configuration location
+      description: 'Required permission "read_configuration"'
+      operationId: getConfiguration
+      parameters:
+        - name: siteId
+          in: query
+          description: Site ID
+          required: true
+          schema:
+            type: string
+        - name: module
+          in: query
+          description: Module name (e.g. studio, engine)
+          required: true
+          schema:
+            type: string
+        - name: location
```

Comment:
In the configuration, this is called `path`. Both `path` and `location` are not exactly right, and they're different.

This is a relative path off of `/config/<module>/`.

Just for us to think about this.

Reply:
Agreed. I didn't want to make too many changes in the configuration.

Reply:
Let's change this to `path.

Reply:
I need to update UI also